documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -856,18 +751,18 @@ where:
 
    Spherical coordinate system used by ``sphere_proj``
 
-..
-   TODO: Move into relevant developer documentation section
 
-   In practice, Horace uses Matlab methods ``cart2sph`` and ``sph2cart`` to convert an array of vectors expressed
-   in Cartesian coordinate system to spherical coordinate system and back.
-   The formulas, used by these methods together with the image of the used coordinate system are provided `on Matlab "cart2sph" help pages <https://uk.mathworks.com/help/matlab/ref/cart2sph.html>`_.
-   Matlab uses ``elevation`` angle which is related to :math:`\theta` angle used by Horace by relation:
+Alternative description of spherical coordinate system may be found on MATLAB help pages.
+Horace uses MATLAB methods ``cart2sph`` and ``sph2cart`` to convert an array of vectors expressed
+in Cartesian coordinate system to spherical coordinate system and back.
+The formulas, used by these methods together with the image of the used coordinate system is provided 
+`on MATLAB "cart2sph" help pages <https://uk.mathworks.com/help/matlab/ref/cart2sph.html>`_.
+MATLAB uses ``elevation`` angle which is related to :math:`\theta` angle used by Horace by relation:
 
        :math:`\theta = 90-elevation`
 
-   ``azimuth`` angle form `Matlab help pages <https://uk.mathworks.com/help/matlab/ref/cart2sph.html>`_
-   is equivalent to Horace :math:`\phi` angle.
+``azimuth`` angle form `MATLAB help pages <https://uk.mathworks.com/help/matlab/ref/cart2sph.html>`_
+is equivalent to Horace :math:`\phi` angle.
```

Comment:
formulas have been removed but I insist on adding reference to actual formulas, used in Horace. Who wants assurance or mor clarity, will go and check for himself